tomcat/src/main/java/camp/nextstep/http/Path.java

```diff
@@ -0,0 +1,47 @@
+package camp.nextstep.http;
+
+import java.util.HashMap;
+
+public class Path {
+
+  private static final String QUERY_STRING_INDICATOR = "?";
+
+  private static final String QUERY_STRING_SEPARATOR = "\\?";
+  private final String urlPath;
+  private final QueryString queryString;
```

Comment:
상수와 상태 값은 구분지어주면 좋습니다
```suggestion
  private static final String QUERY_STRING_INDICATOR = "?";
  private static final String QUERY_STRING_SEPARATOR = "\\?";
  
  private final String urlPath;
  private final QueryString queryString;
```

Reply:
세세한 피드백 감사합니다! 😀 숙지하고 있겠습니다! ㅎㅎ